go.mod

```diff
@@ -9,18 +9,20 @@ require (
 	github.com/cespare/cp v0.1.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/deckarep/golang-set v0.0.0-20180603214616-504e848d77ea
-	github.com/docker/docker v1.4.2-0.20180625184442-8e610b2b55bf
+	github.com/docker/docker v1.6.2
 	github.com/dominant-strategies/bn256 v0.0.0-20220930122411-fbf930a7493d
+	github.com/edsrzf/mmap-go v1.1.0
+	github.com/ethereum/go-ethereum v1.12.0
```

Comment:
Interesting!

Reply:
oh boi... let me get to the bottom of that, haha

Reply:
Fixed! I forgot to run `go mod tidy` since a previous iteration which _did_ depend on `go-ethereum`. Not an issue anymore.